nthprime/lib/nthfibonacci.rb

```diff
@@ -0,0 +1,12 @@
+class NthFibonacci
+  attr_reader :n
```

Comment:
Consider adding an empty line after the attr_reader to separate it from the next method https://github.com/bbatsov/ruby-style-guide